### DISCUSSION TITLE
Why remove the boots from soul elytra?

### DISCUSSION THREAD

**Discussion by CanecaKun** [KyaniteMods/DeeperAndDarker#266] [Question] [answered]
Hello, i watched a mod review and saw the cool elytra with boots and the new armor and dimension and it hyped me to download your mod and I added it to my 1.20.1 modpack, but i got the soul elytra now and it does not do anything, is this a bug?

I read somewhere that you guys "nerfed" it´s boosts but i didn´t see anywhere saying you removed it completely 
Now its just an absurd grind searching for deep dark cities, defeating wardens and all that for a retextured elytra?
I already use other mods like alexs mobs and incendium that have armored elytras so a few armor points is just not worth it without the boosts, is there a way or is it in the plans to bring it back?
Maybe nerfed only boosting every 10 seconds or something so it´s not an infinite boost but more of a utility thing to save rockets?

**Comment by NewJumper** [comment, marked as answer]
in deeper darker 1.2, the boost was removed, but in the next update it is coming back with a cooldown between boosts
